v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/mysql/MySQLConnectionHelper.java

```diff
@@ -29,9 +29,9 @@
 import org.slf4j.LoggerFactory;
 
 /** This is a per Dataflow worker singleton that holds connection pool. */
-public class ConnectionHelper {
+public class MySQLConnectionHelper {
```

Comment:
ConnectionHelper as such is generic, same can be reused for any JDBC connector - only the JDBC URL would change and the connect init SQL. So only those parts should be abstracted out.

Reply:
There will be changes in the ConnectionHelper in case of PostgreSQL , for example it will require schema name and the syntax for `config.setConnectionInitSql( "SET SESSION net_read_timeout=1200")` will also be different

Reply:
All those can be moved to the new ConnectionHelperRequest you created and this can stay a generic JDBCConnectionHelper

Reply:
Thank you!